arm-insights/2015-04-01/swagger/insightsClient.json

```diff
@@ -0,0 +1,1123 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "InsightsClient",
+    "version": "2015-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/microsoft.insights/eventtypes/management/eventcategories": {
+      "get": {
+        "tags": [
+          "Event"
+        ],
+        "operationId": "EventCategories_List",
+        "description": "The list of event categories.",
+        "parameters": [
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2015-04-01",
+            "description": "The api-version to send to the server"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to get the list of event categories",
+            "schema": {
+              "$ref": "#/definitions/EventCategoryCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/microsoft.insights/eventtypes/management/digestEvents": {
+      "get": {
+        "tags": [
+          "Event"
+        ],
+        "operationId": "DigestEvent_List",
+        "description": "The List Digest Event Values operation lists the digest events.",
+        "parameters": [
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2015-04-01",
+            "description": "The api-version to send to the server"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The filter to apply on the operation."
+          },
+          {
+            "name": "$select",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The list of property names to be returned. You can save bandwith by selecting only the properties you need."
```

Comment:
This is what i found from the msdn rest api spec website - https://msdn.microsoft.com/en-us/library/azure/dn931934.aspx. This info should be added in the description.
```
Use the $select to fetch events with only the following properties:
authorization
channels
claims
correlationId
description
eventDataId
eventName
eventSource
eventTimestamp
httpRequest
level
operationId
operationName
properties
resourceGroupName
resourceProviderName
resourceUri
status
submissionTimestamp
subStatus
subscriptionId
```